src/pages/create-course/CreateCourse.tsx

```diff
@@ -10,12 +11,62 @@ import { authRoutes } from '~/router/constants/authRoutes'
 import { ButtonTypeEnum, ButtonVariantEnum } from '~/types'
 
 import { styles } from '~/pages/create-course/CreateCourse.styles'
+import { countActiveOfferFilters } from '~/utils/count-active-filters'
+import { useFilterQuery } from '~/hooks/use-filter-query'
+import { useAppSelector } from '~/hooks/use-redux'
+import { getOpositeRole } from '~/utils/helper-functions'
+import { defaultFilters } from '~/pages/find-offers/FindOffers.constants'
+
+import CourseSearchToolbar from '~/containers/my-courses/course-search-tool-bar/CourseSearchToolbar'
+import CourseCreateDescription from '~/containers/my-courses/course-search-tool-bar/CourseCreateDescription'
 
 const CreateCourse = () => {
   const { t } = useTranslation()
   const navigate = useNavigate()
+
+  const { userRole } = useAppSelector((state) => state.appMain)
+
+  const oppositeRole = getOpositeRole(userRole)
+
+  const { filters, filterQueryActions } = useFilterQuery({
```

Comment:
I guess you don`t need to set filters to query, it should just change state value, after we refactor it with using form

Reply:
may i leave it as it is for now in filters instead of the state? it should not break functionality and so i would be able to refactor this after demo?